bcda/service/config.go

```diff
@@ -119,19 +127,12 @@ func (config *ACOConfig) LookbackTime() time.Time {
 
 	// If we passed our perf year transition, we consider us to be in the new performance year.
 	// Otherwise we are still in the previous performance year.
-	if now.Month() >= config.perfYear.Month() && now.Day() >= config.perfYear.Day() {
+	if now.Month() >= config.perfYear.Month() || (now.Month() == config.perfYear.Month() &&
+		now.Day() >= config.perfYear.Day()) {
```

Comment:
This is the only change.

I forgot I moved codes around so it was easier to look at the data.